app/views/staffing_mailer/staffing_reminder.html.erb

```diff
@@ -1,6 +1,6 @@
 <p>Dear <%= @user.first_name %>,</p>
 
-<p>You are due to be staffing for <%= @staffing.show_title %> at <%= @start_time %></p>
+<p>You are due to be staffing for <%= @staffing.show_title %> at <%= @start_time %>. Make sure to be on time. Please let the Front of House manager know if you can no longer make the slot, or let the DM () or committee rep () know if you run late.</p>
```

Comment:
is this supposed to have a variable in the brackets?

Reply:
Yes, forgot to do that. Will add. Thank you for spotting.